takwimu/templates/profile/profile_detail_takwimu.html

```diff
@@ -2,27 +2,14 @@
 {% load staticfiles sass_tags geotags %}
 
 {% block profile_header_tail %}
-<div class="column-full">
-    <strong>{{ primary_releases.active.name }} {{ primary_releases.active.year }}</strong>
-    {% if primary_releases.other %}
-    <div class="tool-group toggle-sub-group release-list">
-        <a>Change release<i class="fa fa-chevron-circle-down"></i></a>
-        <ul class="sub-group" style="display: none;">
-        {% for release in primary_releases.other %}
-            <li><a href="{{ request.path }}?release={{release.year}}">{{ release.name }} {{ release.year }}</a></li>
-        {% endfor %}
-        </ul>
-    </div>
-    {% endif %}
-</div>
 {% endblock %}
 
 
 
 {% block profile_detail %}
 
 {# DEMOGRAPHICS #}
-{% if not demographics.is_missing %}
+{% if not demographics.is_missing or not worldbank.is_missing %}
```

Comment:
At section level like this, I think it's best to check for specific `worldbank` data sets e.g. `.. or not worldbank.youth_unemployment.is_missing or not worldbank.youth...` , otherwise we might end up with empty sections just because there is `worlbank` data for other sections